src/rpm/headers/signatures.rs

```diff
@@ -61,7 +61,11 @@ impl SignatureHeaderBuilder {
             // need to base64-encode the raw bytes of the signatures
             for sig_bytes in &self.openpgp_signatures {
                 let signature = Verifier::parse_signature(sig_bytes)?;
-                let tag = match signature.config.pub_alg {
+                let tag = match signature
+                    .config()
```

Comment:
Does the non-existence of the `config()` return value *actually* imply missing version or is it slightly different?

Reply:
This is my understanding yes:
https://github.com/rpgp/rpgp/blob/80977cafbb01bdb675883a96e06a4e678cf8fa73/src/packet/signature/types.rs#L232-L237

https://github.com/rpgp/rpgp/pull/520